.eslintrc.json

```diff
@@ -54,7 +54,9 @@
             "semi": false,
             "arrowParens": "avoid"
           }
-        ]
+        ],
+        "no-shadow": "off",
```

Comment:
This allows typescript enums to be used without the IDE highlighting a linting error